cmd/ack-generate/command/crossplane.go

```diff
@@ -80,7 +80,7 @@ func generateCrossplane(_ *cobra.Command, args []string) error {
 		cfgPath = ""
 	}
 	m, err := ackmodel.New(
-		sdkAPI, svcAlias, optGenVersion, cfgPath, cpgenerate.DefaultConfig,
+		sdkAPI, svcAlias, optModelName, optGenVersion, cfgPath, cpgenerate.DefaultConfig,
```

Comment:
If we move service model name override to being a field in the `generator.yaml` file, we won't need to change any of these lines.